package.json

```diff
@@ -14,7 +14,7 @@
     "event-kit": "^1.0.0",
     "fs-plus": "^2.3.0",
     "minimatch": "~0.3.0",
-    "pathwatcher": "^2.0.12",
+    "pathwatcher": "~2.0.12",
```

Comment:
The version that was downloaded was causing an incompatible error to be thrown by atom.

See #304